src/Tests/dotnet-new.Tests/Approvals/DotnetNewListTests.BasicTest_WhenLegacyCommandIsUsed_common.Linux.verified.txt

```diff
@@ -1,41 +1,41 @@
 ﻿Warning: use of 'dotnet new --list' is deprecated. Use 'dotnet new list' instead.
-For more information, run:
```

Comment:
related question - should we put an editorconfig file in the `Approvals` directory that tells editors to not trim trailing whitespace for  all `.txt` files?

Reply:
It can be also done via globbing pattern in repo one.